packages/create-sitecore-jss/src/templates/angular/package.json

```
@@ -34,7 +34,9 @@
     "build:server": "cross-env-shell ng run <%- appName %>:server:production --output-path=$npm_package_config_buildArtifactsPath/server --output-hashing=none --bundle-dependencies true",
     "postbuild:server": "move-cli ./dist/server/main.js ./dist/server.bundle.js && del-cli ./dist/server",
     "bootstrap": "ts-node --project src/tsconfig.webpack-server.json scripts/bootstrap.ts",
-    "graphql:update": "ts-node --project src/tsconfig.webpack-server.json ./scripts/update-graphql-fragment-data.ts"
+    "graphql:update": "ts-node --project src/tsconfig.webpack-server.json ./scripts/update-graphql-fragment-data.ts",
+    "install-pre-commit-hook": "node ./scripts/install-pre-commit-hook.js",
+    "pre-commit": "yarn lint --fix"
```

Suggest to remove the dash to follow [npm scripts lifecycle](https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts) naming convention (just in case we want to come up with "commit" or "postcommit" actions).
```suggestion
    "precommit": "yarn lint --fix"
```